spec/lib/object_serializer_inheritance_spec.rb

```diff
@@ -51,6 +51,15 @@ class UserSerializer
     has_one :photo
   end
 
+  class Address
```

Comment:
These classes were referenced but not actually defined.  The serializer classes are now required.